tensorflow/python/util/compat.py

```diff
@@ -111,7 +111,8 @@ def as_str_any(value):
 
 @tf_export('compat.path_to_str')
 def path_to_str(path):
-  """Returns the file system path representation of a `PathLike` object, else as it is.
+  """Returns the file system path representation of a `PathLike` object, 
+  else as it is.
```

Comment:
Don't change this line. I know it is already too long but first phrase in docstring should always be on a single line and followed by empty space (please run linter locally and check the warnings issued on the lines you're changing).